shared/templates/static/ansible/require_smb_client_signing.yml

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+- name: Check if /etc/samba/smb.conf exists
+  stat:
+    path: /etc/samba/smb.conf
+  register: st_smb
```

Comment:
all items related to the rule need @ANSIBLE_TAGS@

That way if you just select one rule by its tag it will work. Otherwise it would skip the register tag and then use undeclared variables.